tasks/utils.yaml

```diff
@@ -64,3 +64,158 @@ tasks:
               --from-literal=password=${PASSWORD} \
               -n keycloak
           fi
+  - name: eks-coredns-setup
+    actions:
+      - description: Setup Custom ConfigMap for Core DNS
+        cmd: |
+          uds zarf tools kubectl apply -f - <<EOF
+          apiVersion: v1
+          data:
+            Corefile: |
+              .:53 {
+                  errors
+                  health {
+                      lameduck 5s
+                    }
+                  ready
+                  kubernetes cluster.local in-addr.arpa ip6.arpa {
+                    pods insecure
+                    fallthrough in-addr.arpa ip6.arpa
+                  }
+                  prometheus :9153
+                  forward . /etc/resolv.conf
+                  cache 30
+                  loop
+                  reload
+                  loadbalance
+                  rewrite stop {
+                    name regex (.*\.admin\.uds\.dev) admin-ingressgateway.istio-admin-gateway.svc.cluster.local answer auto
+                    }
+                  rewrite stop {
+                  name regex (.*\.uds\.dev) tenant-ingressgateway.istio-tenant-gateway.svc.cluster.local answer auto
+                    }
+              }
+          kind: ConfigMap
+          metadata:
+            name: coredns
+            namespace: kube-system
+          EOF
+            uds zarf tools kubectl -n kube-system rollout restart deployment coredns
+  - name: aks-coredns-setup
+    actions:
+      - description: Setup Custom ConfigMap for Core DNS
+        cmd: |
+          uds zarf tools kubectl apply -f - <<EOF
+          apiVersion: v1
+          data:
+            uds.override: |
+              rewrite stop {
+                name regex (.*\.admin\.uds\.dev) admin-ingressgateway.istio-admin-gateway.svc.cluster.local answer auto
+              }
+              rewrite stop {
+                name regex (.*\.uds\.dev) tenant-ingressgateway.istio-tenant-gateway.svc.cluster.local answer auto
+              }
+          kind: ConfigMap
+          metadata:
+            name: coredns-custom
+            namespace: kube-system
+          EOF
+            uds zarf tools kubectl -n kube-system rollout restart deployment coredns
+  - name: rke2-allow-prom-kube-dns
+    actions:
+      - description: Create NetworkPolicy to allow Prometheus to scrape kube-dns
+        cmd: |
+          uds zarf tools kubectl apply -f - <<EOF
+          apiVersion: networking.k8s.io/v1
+          kind: NetworkPolicy
+          metadata:
+            name: allow-prometheus-to-kube-dns
+            namespace: kube-system
```

Comment:
Don't have to do this now, but still worth evaluating if this should be a conditional netpol in core?